src/Core/Traits/HasAttributes.php

```diff
@@ -45,7 +47,7 @@ public function attribute($handle, $channel = null, $locale = null)
         }
 
         if (empty($this->attribute_data[$handle][$channel])) {
-            $defaultChannel = GetCandy::channels()->getDefaultRecord();
+            $defaultChannel = FetchDefaultChannel::run();
```

Comment:
`FetchDefaultChannel` is that the right name? Or should it be more along the lines of `FetchSelectedChannel`? I'm not too sure what this does haha

Reply:
Oh and we talked about the Channel closure idea.

Reply:
This doesn't send back the current channel, just the one that's been marked as `default` in the database.

Reply:
OK